benchmarks/serialization_libraries/compare.sh

```diff
@@ -0,0 +1,222 @@
+#         },                                                              },
+#         {                                                               {
+#           "id": "5",                                                      "id": "5",
+#           "type": "posts"                                                 "type": "posts"
+#         }                                                              }
+#       ]                                                               ]
+#     }                                                               }
+#   },                                                              },
+#   "type": "users"                                                 "type": "users"
+# }                                                               }
+# -------------------------------------included posts
+# {                                                               {
+#   "attributes": {                                                 "attributes": {
+#     "body": "awesome content",                                      "body": "awesome content",
+#     "created_at": "2017-07-01 05:00:00 UTC",                        "created_at": "2017-07-01 05:00:00 UTC",
+#     "title": "Some Post",                                           "title": "Some Post",
+#     "updated_at": "2017-07-01 05:00:00 UTC"                         "updated_at": "2017-07-01 05:00:00 UTC"
+#   },                                                              },
+#   "id": "2",                                                      "id": "2",
+#   "relationships": {                                              "relationships": {
+#     "comments": {                                                   "comments": {
+#       "data": [                                                       "data": [
+#         {                                                               {
+#           "id": "3",                                                      "id": "3",
+#           "type": "comments"                                              "type": "comments"
+#         },                                                              },
+#         {                                                               {
+#           "id": "4",                                                      "id": "4",
+#           "type": "comments"                                              "type": "comments"
+#         }                                                               }
+#       ]                                                               ]
+#     },                                                              },
+#     "user": {                                                       "user": {
+#       "meta": {                                               |       "data": {
+#         "included": false                                     |         "id": "1",
+#                                                               >         "type": "users"
```

Comment:
jsonapi-rb excludes the posts.relationships.user.data since posts are included via user.posts